schema/project-level/codelists/documentType.csv

```diff
@@ -61,3 +61,4 @@ technicalAuditReport,Technical audit report,Documentation of checks that constru
 qualityAssuranceReport,Quality assurance report,"Documentation identifying deficiencies in designs, specifications, standards, methods, procedures and practices, and identifying best practices.",OC4IDS
 incorporationCertificate,Certificate of incorporation,A document issued by the relevant government registry as confirmation of the due incorporation and valid existence of the organization.,OC4IDS
 contractAmendment,Contract amendment,A legal document that changes or adds to the terms of an existing contract.,OC4IDS
+designReport,Project design report,"A record of the justification for the design approach adopted in the project, typically issued before the works start by the consultant or firm that will supervise the works.",OC4IDS
```

Comment:
I find this order easier to read. Alternatively, we can just add a comma after "start" to make it clearer what "by …" is modifying.

```suggestion
designReport,Project design report,"A record of the justification for the design approach adopted in the project, typically issued by the consultant or firm that will supervise the works before the works start.",OC4IDS
```

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/289/commits/0fb1a4db2cb94ebd9d69f311ccfde6b7ed278fb4